swig/cpp/examples/yang/ietf-interfaces.data.xml

```diff
@@ -0,0 +1,30 @@
+<interfaces xmlns="urn:ietf:params:xml:ns:yang:ietf-interfaces">
```

Comment:
There is no need for adding the YANG files and data files already present in /examples/yang/ directory